project/accounts/urls.py

```diff
@@ -35,3 +38,25 @@
     ),
     path('setup/', ProfileSetupView.as_view(), name="accounts_profile_setup"),
 ]
+
+
+# REST endpoints
+router = DefaultRouter(trailing_slash=False)
```

Comment:
Keep in mind issue #1030 

What is the default path/prefix for this `router`? At what endpoint does it server the JSON docs?

https://www.django-rest-framework.org/api-guide/routers/#defaultrouter

Reply:
My bad. I need to remove `DefaultRouter` from `accounts.urls.py` because we'll create the global api router. Right?